### PR TITLE
Allow testdriver to return values

### PR DESCRIPTION
**Pull request by kereliuk**
This is tracked by #10716

Still need to write an infrastructure test and write some documentation.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/11080)

### CODE REVIEW

**Review by gsnedders — COMMENTED** [review]
Modulo these two nits, this looks fine

**Comment by gsnedders** [comment]
@kereliuk How did you plan on testing this? This seems like it shouldn't be testable in isolation? Docs would be good, though.

**Comment by gsnedders** [comment]
@kereliuk ^^

File: tools/wptrunner/wptrunner/executors/base.py [hunk 1]
```diff
@@ -531,14 +532,18 @@ def process_action(self, url, payload):
             except KeyError:
                 raise ValueError("Unknown action %s" % action)
             try:
-                action_handler(payload)
+                value = action_handler(payload)
+                self.logger.debug("Recieved value: %s" % value)
```
Comment (jugglinmike) [inline tools/wptrunner/wptrunner/executors/base.py]: Typo: "Recieved" -> "Received"

**Comment by foolip** [comment]
Someone else will have to take this over, so I'll close and point here from the issue, for when we prioritize the issue..

**Pull request closed without merging**